src/core/friendly_errors/sketch_verifier.js

```diff
@@ -1,11 +1,64 @@
 import * as acorn from 'acorn';
 import * as walk from 'acorn-walk';
+import * as constants from '../constants';
 
 /**
  * @for p5
  * @requires core
  */
 function sketchVerifier(p5, fn) {
+  // List of functions to ignore as they either are meant to be re-defined or
+  // generate false positive outputs.
+  const ignoreFunction = [
+    'setup',
+    'draw',
+    'preload',
+    'deviceMoved',
+    'deviceTurned',
+    'deviceShaken',
+    'doubleClicked',
+    'mousePressed',
+    'mouseReleased',
+    'mouseMoved',
+    'mouseDragged',
+    'mouseClicked',
+    'mouseWheel',
+    'touchStarted',
+    'touchMoved',
+    'touchEnded',
+    'keyPressed',
+    'keyReleased',
+    'keyTyped',
+    'windowResized',
+    'name',
+    'parent',
+    'toString',
+    'print',
+    'stop',
+    'onended'
+  ];
+
+  // Mapping names of p5 types to their constructor functions.
+  // p5Constructors:
+  //   - Color: f()
+  //   - Graphics: f()
+  //   - Vector: f()
+  // and so on.
+  const p5Constructors = {};
+
+  fn.loadP5Constructors = function () {
+    // Make a list of all p5 classes to be used for argument validation
+    // This must be done only when everything has loaded otherwise we get
+    // an empty array
+    for (let key of Object.keys(p5)) {
+      // Get a list of all constructors in p5. They are functions whose names
+      // start with a capital letter
+      if (typeof p5[key] === 'function' && key[0] !== key[0].toLowerCase()) {
+        p5Constructors[key] = p5[key];
+      }
+    }
+  }
```

Comment:
If this function is internal to this module only, ie. not called by other parts of the p5.js code base via `this.loadP5Constructors()` this should probably be defined simply as a scoped function:

```js
const loadP5Constructors = () => {
}
```